apps/pom.xml

```diff
@@ -200,6 +201,31 @@
 				<artifactId>metrics-jvm</artifactId>
 				<version>${metrics.version}</version>
 			</dependency>
+			<dependency>
+				<!-- Used to collect metrics in a registry that can be transmitted to CloudWatch efficiently. -->
+				<groupId>io.micrometer</groupId>
+				<artifactId>micrometer-core</artifactId>
+				<version>${micrometer.version}</version>
+			</dependency>
+			<dependency>
+				<!-- Used to transmit metrics to JMX for local testing. -->
+				<groupId>io.micrometer</groupId>
+				<artifactId>micrometer-registry-jmx</artifactId>
+				<version>${micrometer.version}</version>
+			</dependency>
+			<dependency>
+				<!-- Used to transmit metrics to CloudWatch for alerts and dashboards. -->
+				<groupId>io.micrometer</groupId>
+				<artifactId>micrometer-registry-cloudwatch</artifactId>
+				<version>${micrometer.version}</version>
+			</dependency>
+			<dependency>
+				<!-- Used to transmit metrics to NewRelic for alerts and dashboards. -->
+				<groupId>io.micrometer</groupId>
+				<artifactId>micrometer-registry-newrelic</artifactId>
+				<version>${micrometer.version}</version>
+			</dependency>
```

Comment:
👋 Hey, @StewGoin can you provide any input on including this suite of micrometer libraries? Thanks!

Reply:
No concerns from me on using this here as well.